traitsui/examples/demo/Extras/Image_editor_demo.py

```diff
@@ -15,20 +15,18 @@
 from traits.api \
     import HasTraits, Str
 
-from traitsui.api \
-    import View, VGroup, Item
+import traitsui.api
```

Comment:
This added line is not needed any more (neither is the `import traits` at the top?)